projects/jsf-launcher/src/app/schemaV2.json

```diff
@@ -969,6 +969,39 @@
           }
         }
       }
+    },
+    "templateTest": {
+      "type": "object",
+      "name": "Template Test",
```

Comment:
Can you remove the word "Test" so that it is more consistent with the other tab names?